src/main/java/org/openstreetmap/atlas/geography/MultiPolygon.java

```diff
@@ -113,6 +120,18 @@ public GeoJsonObject asGeoJsonFeatureCollection()
         return new MultiIterable<>(outers, inners);
     }
 
+    /**
+     * @return Optional of {@link Polygon} representation if possible
+     */
+    public Optional<Polygon> asSimplePolygon()
+    {
+        if (this.isSimplePolygon())
+        {
+            return outers().stream().findFirst();
+        }
```

Comment:
Maybe add a WARN log here to say "trying to save a complex multipolygon as a simple polygon"

Reply:
What context do you want the logger warning?